FAQ.md

```diff
@@ -34,8 +34,7 @@ Summarize the question and quote the reply, linking back to the original comment
 * [What semver version should I use?](#what-semver-version-should-i-use)
 * [Is it OK to make backwards-incompatible changes now?](#is-it-ok-to-make-backwards-incompatible-changes-now)
 * [My dependers don't use `dep` yet. What should I do?](#my-dependers-dont-use-dep-yet-what-should-i-do)
-
-___
+* [How do I configure a dependency that doesn't tag its releases?](#how-do-I-configure-a-dependency-that-doesnt-tag-its-releases)
```

Comment:
One more change, I swear! Thank you for sticking with me. 😁 

The link isn't scrolling down the page to the right section because of the upper-case "I" in the url. It should be this `#how-do-i-configure-a-dependency-that-doesnt-tag-its-releases`.

I tried to "help" before by adding a commit to your PR but I think that made it harder... Sorry about that!